azure-spring-cloud-maven-plugin/src/main/java/com/microsoft/azure/maven/spring/spring/SpringAppClient.java

```diff
@@ -23,8 +23,9 @@
 
 public class SpringAppClient extends AbstractSpringClient {
 
-    protected static final String DEFAULT_DEPLOYMENT_NAME = "init";
     protected static final int DEFAULT_PERSISTENT_DISK_SIZE = 50;
+    protected static final String DEFAULT_DEPLOYMENT_NAME = "default";
```

Comment:
move DEFAULT-XXX to class PersistentDisk

Reply:
'PersistentDisk' is the class in auto generated SDK, shall we change it?